ReactSkia/components/RSkComponentImage.cpp

```diff
@@ -75,7 +76,15 @@ void RSkComponentImage::OnPaint(SkCanvas *canvas) {
       /*TODO Not applied any shadow for solid Image when resizeMode is "Repeat"*/
```

Comment:
IS this to do or a Bug: Please confirm. IF bug: remove TO do statement from here and record the Bug seperately